#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_sphinx.conf.dpatch by  <monty@inaugust.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Changed default locations to match debian locations.

@DPATCH@
--- ./sphinx.conf.in~	2009-10-09 09:32:51.000000000 +0000
+++ ./sphinx.conf.in	2009-10-09 09:36:17.000000000 +0000
@@ -216,14 +216,14 @@
 	# shell command to invoke xmlpipe stream producer
 	# mandatory
 	#
-	# xmlpipe_command	= cat @CONFDIR@/test.xml
+	# xmlpipe_command	= cat @localstatedir@/test.xml
 
 	#####################################################################
 	## xmlpipe2 settings
 	#####################################################################
 
 	# type				= xmlpipe2
-	# xmlpipe_command	= cat @CONFDIR@/test2.xml
+	# xmlpipe_command	= cat @localstatedir@/test2.xml
 
 
 	# xmlpipe2 field declaration
@@ -277,7 +277,7 @@
 
 	# index files path and file name, without extension
 	# mandatory, path must be writable, extensions will be auto-appended
-	path			= @CONFDIR@/data/test1
+	path			= @localstatedir@/data/test1
 
 	# document attribute values (docinfo) storage mode
 	# optional, default is 'extern'
@@ -312,13 +312,13 @@
 	# optional, default is empty
 	# contents are plain text, charset_table and stemming are both applied
 	#
-	# stopwords			= @CONFDIR@/data/stopwords.txt
+	# stopwords			= @CONFDIR@/stopwords.txt
 
 
 	# wordforms file, in "mapfrom > mapto" plain text format
 	# optional, default is empty
 	#
-	# wordforms			= @CONFDIR@/data/wordforms.txt
+	# wordforms			= @CONFDIR@/wordforms.txt
 
 
 	# tokenizing exceptions file
@@ -327,7 +327,7 @@
 	# plain text, case sensitive, space insensitive in map-from part
 	# one "Map Several Words => ToASingleOne" entry per line
 	#
-	# exceptions		= @CONFDIR@/data/exceptions.txt
+	# exceptions		= @CONFDIR@/exceptions.txt
 
 
 	# minimum indexed word length
@@ -480,7 +480,7 @@
 # and may then be overridden in this index definition
 index test1stemmed : test1
 {
-	path			= @CONFDIR@/data/test1stemmed
+	path			= @localstatedir@/data/test1stemmed
 	morphology		= stem_en
 }
 
@@ -568,19 +568,19 @@
 	# multi-value, multiple listen points are allowed
 	# optional, default is 0.0.0.0:3312 (listen on all interfaces, port 3312)
 	#
-	# listen				= 127.0.0.1
+	listen				= 127.0.0.1
 	# listen				= 192.168.0.1:3312
 	# listen				= 3312
-	# listen				= /var/run/searchd.sock
+	# listen				= /var/run/sphinxsearch/searchd.sock
 
 
 	# log file, searchd run info is logged here
 	# optional, default is 'searchd.log'
-	log					= @CONFDIR@/log/searchd.log
+	log					= /var/log/sphinxsearch/searchd.log
 
 	# query log file, all search queries are logged here
 	# optional, default is empty (do not log queries)
-	query_log			= @CONFDIR@/log/query.log
+	query_log			= /var/log/sphinxsearch/query.log
 
 	# client read timeout, seconds
 	# optional, default is 5
@@ -596,7 +596,7 @@
 
 	# PID file, searchd process ID file name
 	# mandatory
-	pid_file			= @CONFDIR@/log/searchd.pid
+	pid_file			= /var/run/sphinxsearch/searchd.pid
 
 	# max amount of matches the daemon ever keeps in RAM, per-index
 	# WARNING, THERE'S ALSO PER-QUERY LIMIT, SEE SetLimits() API CALL
@@ -642,7 +642,7 @@
 	# searchd will (try to) log crashed query to 'crash_log_path.PID' file
 	# optional, default is empty (do not create crash logs)
 	#
-	# crash_log_path		= @CONFDIR@/log/crash
+	# crash_log_path		= /var/log/sphinxsearch/crash
 
 
 	# max allowed per-query filter count
